src/Microsoft.SqlTools.ServiceLayer/EditData/Contracts/EditCell.cs

```diff
@@ -0,0 +1,39 @@
+﻿//
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+//
+
+using Microsoft.SqlTools.ServiceLayer.QueryExecution.Contracts;
+
+namespace Microsoft.SqlTools.ServiceLayer.EditData.Contracts
+{
+    /// <summary>
+    /// Cell that wraps info from <see cref="DbCellValue"/> for edit purposes
+    /// </summary>
+    public class EditCell : DbCellValue
+    {
+        /// <summary>
+        /// Default, parameterless constructor to make sure that JSON serializing is happy
+        /// </summary>
+        public EditCell() {}
+
+        /// <summary>
+        /// Constructs a new EditCell based on a DbCellValue
+        /// </summary>
+        /// <param name="dbCellValue">The DbCellValue that will be enhanced</param>
+        /// <param name="isDirty">Whether or not the edit cell is dirty</param>
+        public EditCell(DbCellValue dbCellValue, bool isDirty)
+        {
+            IsDirty = isDirty;
+
+            DisplayValue = dbCellValue.DisplayValue;
```

Comment:
2 points:
- Please validate dbCellValue arg is not null
- Often we implement a `CopyTo` method on a class like DbCellValue if you want to copy over some data from it to this new subclass. You could put that in and call it from this constructor, meaning that the parent class will know what should be copied instead of this subclass knowing.